src/components/cli-tutor-mode/CliTutorMode.js

```diff
@@ -0,0 +1,110 @@
+import React, { Component, Fragment } from 'react'
+import PropTypes from 'prop-types'
+import { connect } from 'redux-bundler-react'
+
+// Components
+import { Modal, ModalBody, ModalActions } from '../modal/Modal'
+import CopyIcon from '../../icons/CopyIcon'
+import Button from '../button/Button'
+import Overlay from '../overlay/Overlay'
+import Shell from '../shell/Shell'
+import StrokeDownload from '../../icons/StrokeDownload'
+
+export const CliTutorialModal = ({ command, t, onLeave, className, downloadConfig, ...props }) => {
+  const onClickCopyToClipboard = async (command) => {
+    await navigator.clipboard.writeText(command)
+  }
+
+  return (
+    <Modal {...props} className={className} onCancel={onLeave} style={{ maxWidth: '40em' }}>
+      <ModalBody icon={CopyIcon}>
+        <p className='charcoal w-80 center'>
+          {t('cli.description')}
+        </p>
+        <p className='charcoal-muted w-90 center'>
+          {command.includes('ipfs config replace') ? t('cli.extraNotes') : ''}
+        </p>
+        <div>
+          <Shell title="Shell">
+            <code className='db'><b className='no-select'>$ </b>{command}</code>
+          </Shell>
+        </div>
+      </ModalBody>
+
+      <ModalActions>
+        <div>
+          <Button className='ma2 tc' bg='bg-gray' onClick={onLeave}>{t('actions.close')}</Button>
+        </div>
+        <div className='flex items-center'>
+          {
+            command && command.includes('ipfs config replace')
+              ? <StrokeDownload
+                onClick={downloadConfig}
+                style={{ height: '28px', transform: 'scale(1.5)', verticalAlign: 'bottom', color: 'dodgerblue' }}
+                className='dib fill-current-color ph2 glow o-80 pointer'
+              /> : <div />
+          }
+          <Button className='ma2 tc' onClick={() => onClickCopyToClipboard(command)}>
+            {t('actions.copyCommand')}
+          </Button>
+        </div>
+      </ModalActions>
+    </Modal>
+  )
+}
+
+class CliTutorMode extends Component {
```

Comment:
You're using a functional component above and here you used a class component. Can you make them equal? (I would advise choosing functional)

Reply:
I've Converted this into a functional component :)